tests/pytest_tests/system_tests/test_artifacts/test_data_types.py

```diff
@@ -66,6 +69,144 @@ def test_log_dataframe(user, test_settings):
     assert len(run.logged_artifacts()) == 1
 
 
+def test_log_nested_cc(user, test_settings, relay_server, inject_graphql_response):
+    server_info_response = inject_graphql_response(
```

Comment:
why do you need this?

Reply:
@dmitryduev, you are right, I don't. Fixed.